From 32602f42c869d69ae35db14d7d89c97f89f08a95 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <sunnyflunk@gmail.com>
Date: Tue, 27 Sep 2016 14:25:26 +1000
Subject: [PATCH] Use Solus defaults path

Make the package stateless by not shipping file in /etc

---
 example/CMakeLists.txt    | 2 +-
 ksysguardd/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/example/CMakeLists.txt b/example/CMakeLists.txt
index a0150e5..617a382 100644
--- a/example/CMakeLists.txt
+++ b/example/CMakeLists.txt
@@ -1,2 +1,2 @@
-install( FILES ksysguarddrc  DESTINATION  ${KDE_INSTALL_FULL_SYSCONFDIR} )
+install( FILES ksysguarddrc  DESTINATION  ${KDE_INSTALL_FULL_CONFDIR}/defaults/ksysguard )
 install( FILES ksysguard.notifyrc DESTINATION  ${KNOTIFYRC_INSTALL_DIR} )
diff --git a/ksysguardd/CMakeLists.txt b/ksysguardd/CMakeLists.txt
index aa9d06d..6d16edd 100644
--- a/ksysguardd/CMakeLists.txt
+++ b/ksysguardd/CMakeLists.txt
@@ -3,7 +3,7 @@ include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/CContLib/ )
 # Laurent: Verify that we install it into (kdeprefix)/etc/ and not into /etc
 # otherwise it's necessary to change it.
 # into kde 3.x it was installed into /etc/
-add_definitions( -DKSYSGUARDDRCFILE="${KDE_INSTALL_FULL_SYSCONFDIR}/ksysguarddrc" )
+add_definitions( -DKSYSGUARDDRCFILE="${KDE_INSTALL_FULL_CONFDIR}/defaults/ksysguard/ksysguarddrc" )

 add_definitions(-DOSTYPE_${CMAKE_SYSTEM_NAME})

--
2.10.0
